Specs/DateTime/English/DateTimeModel.json

```diff
@@ -3962,14 +3962,9 @@
         "Resolution": {
           "values": [
             {
-              "timex": "XXXX-06-15",
+              "timex": "2018-06-15",
```

Comment:
Why this change? This is incorrect.
The original sentence doesn't force 2018. Why is the timex modified??